cfme/cloud/tenant.py

```diff
@@ -49,7 +49,7 @@ class relationships(Accordion):     # noqa
 
 class TenantEntities(BaseEntitiesView):
     """The entities on the main list page"""
-    table = Table('//div[@id="list_grid"]//table')
+    table = Table('//div[@id="gtl_div"]//table')
```

Comment:
**REQURIED**
What about 5.8? Is it changed there as well.

Reply:
This locator covers all 3 versions